core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/LandingZoneCreationStatusRunner.scala

```diff
@@ -0,0 +1,115 @@
+package org.broadinstitute.dsde.rawls.monitor.workspace.runners
+
+import bio.terra.workspace.model.JobReport
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.rawls.billing.BillingRepository
+import org.broadinstitute.dsde.rawls.dataaccess.{GoogleServicesDAO, SamDAO}
+import org.broadinstitute.dsde.rawls.dataaccess.slick.WorkspaceManagerResourceMonitorRecord.{
+  Complete,
+  Incomplete,
+  JobStatus,
+  JobType
+}
+import org.broadinstitute.dsde.rawls.dataaccess.slick.WorkspaceManagerResourceMonitorRecord.JobType.JobType
+import org.broadinstitute.dsde.rawls.dataaccess.slick.{
+  WorkspaceManagerResourceJobRunner,
+  WorkspaceManagerResourceMonitorRecord
+}
+import org.broadinstitute.dsde.rawls.dataaccess.workspacemanager.WorkspaceManagerDAO
+import org.broadinstitute.dsde.rawls.model.{CreationStatuses, RawlsBillingProjectName, RawlsRequestContext}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+
+class LandingZoneCreationStatusRunner(
+  samDAO: SamDAO,
+  workspaceManagerDAO: WorkspaceManagerDAO,
+  billingRepository: BillingRepository,
+  gcsDAO: GoogleServicesDAO
+) extends WorkspaceManagerResourceJobRunner
+    with LazyLogging {
+
+  override val jobType: JobType = JobType.AzureLandingZoneResult
+
+  override def run(
+    job: WorkspaceManagerResourceMonitorRecord
+  )(implicit executionContext: ExecutionContext): Future[JobStatus] = {
+    val billingProjectName = job.billingProjectId match {
+      case Some(name) => RawlsBillingProjectName(name)
+      case None =>
+        logger.error(
+          s"Job to monitor AzureLandingZoneResult created with id ${job.jobControlId} but no billing project set"
+        )
+        return Future.successful(Complete) // nothing more this runner can do with it
+    }
+    val userEmail = job.userEmail match {
+      case Some(email) => email
+      case None =>
+        logger.error(
+          s"Job to monitor AzureLandingZoneResult for billing project $billingProjectName created with id ${job.jobControlId} but no user email set"
```

Comment:
Should this be `${billingProjectName.value}` (does it matter)?

Reply:
The string value of the project includes its object type, so it reads: `RawlsBillingProject(projectName)`.
So I removed the part before that so it will read a bit cleaner.
I figure having the actual object name in the message might make it _slightly_ more helpful